### PR TITLE
Tlc

### PR DESCRIPTION
**Pull request by aaronbean**

### CODE REVIEW

File: roles/deployments-fpp/defaults/main.yml [hunk 1]
```diff
@@ -14,3 +14,5 @@ memsb_version: master
 myifsta_version: master
 nodebb_playbook_version: master
 oscommerce_playbook_version: master
```
Comment (martin-williams) [inline roles/deployments-fpp/defaults/main.yml]: Do we still need this?

Comment (aaronbean) [inline roles/deployments-fpp/defaults/main.yml]: we probably don't, but I guess it makes it flexible and would take more work to remove than leave

**Comment by martin-williams** [comment]
LGTM!

**Comment by tyscorp** [comment]
https://i.imgur.com/wX1ssgd.png

**Pull request merged**